ci/tasks/upgrade-stemcell.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+set -e
+
+dir=`dirname "$0"`
+source ${dir}/utils.sh
+
+base=$( cd "$( dirname "$( dirname "$0" )")"/.. && pwd )
+base_gopath=$( cd $base/../../../.. && pwd )
+go version
+export GOPATH=$base_gopath:$GOPATH
+echo "GOPATH=" $GOPATH
+
+print_title "INSTALL BOSH CLI..."
+echo "installing bosh CLI"
+gem install bosh_cli --no-ri --no-rdo c
```

Comment:
@zhanggbj get it here: https://github.com/cloudfoundry/bosh-cli

It's good to have it on your machine and start trying it. Best,

Max